src/WebJobs.Script/Rpc/ILanguageWorkerProcessManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public interface ILanguageWorkerProcessManager : IDisposable
```

Comment:
This shouldn't be `IDisposable'. The implementation can implement `IDisposable` if needed.

Reply:
Will fix this in followup PR